lambdas/meetupcom/handlers/transformer.js

```diff
@@ -0,0 +1,202 @@
+"use strict";
+
+const path = require("path");
+const striptags = require("striptags");
+const { getFromS3, getListFromS3 } = require("aws-lambda-data-utils");
+const { validate } = require("jsonschema");
+const eventSchema = require("./schemas/event-schema");
+const { buckets } = require("../config");
+const { uploadTo } = require("../utils");
+
+const meetupGroupsFilePrefix = 'meetupcom-groups__';
+const meetupGroupsDirectory = 'groups-events';
+
+const getDescriptionFrom = function ({ plain_text_no_images_description: strippedDescription, description }) {
+  if (strippedDescription) return strippedDescription;
+  return striptags(striptags(description, ['p', 'div']), [], '\n').trim();
+};
+
+const getChargeFrom = function ({ fee }) {
+  if (!fee) return { is_free: true };
+
+  return {
+    is_free: false,
+    cost: {
+      currency: fee.currency.trim(),
+      value: fee.amount
+    }
+  }
+};
+
+const getVenueFrom = function ({ venue, group }, defaults) {
+  if (!venue) {
+    return {
+      name: "No Venue",
+      latitude: `${group.lat}`.trim(),
+      longitude: `${group.lon}`.trim(),
+      address: defaults.localized_location.trim(),
+      city: defaults.city.trim(),
+      country: defaults.country.trim().toUpperCase()
```

Comment:
Default values for when the venue data isn't available - this will just use the groups location with the venue name "No Venue"